app/controllers/admin/tutorials_controller.rb

```diff
@@ -38,11 +39,35 @@ def tutorial_params
     params.require(:tutorial).permit(:tag_list)
   end
 
-  def results
-    YoutubePlaylistResults.new(params[:playlist_id])
+  def valid_thumbnail?(thumbnail)
+    thumbnail = thumbnail.downcase
+    valid_thumbnail1 = 'http://img.youtube.com'
+    valid_thumbnail2 = 'https://img.youtube.com'
+    valid_thumbnail3 = 'img.youtube.com'
```

Comment:
if you enter any of those addresses you will receive at least a broken youtube video image

Reply:
![](https://media2.giphy.com/media/118YiuxwlUwMEM/giphy.gif?cid=5a38a5a267f6f0e9e4efbbd93bb5c9935c088c55520369d2&rid=giphy.gif)